qa/workunits/rbd/rbd_mirror_stress.sh

```diff
@@ -1,412 +1,149 @@
 #!/bin/bash
```

Comment:
Minor: it looks like you have removed dependency on bash $RANDOM, so may be change this to /bin/sh?